### DISCUSSION TITLE
Import Unmanaged VM fails due to no matching network found (vCenter)

### DISCUSSION THREAD

**Discussion by antoniorsys** [apache/cloudstack#10211] [Q&A]
##### ISSUE TYPE

 * Other

##### COMPONENT NAME
~~~
UI
~~~

##### CLOUDSTACK VERSION
~~~
 4.18.1.0
~~~

##### CONFIGURATION
~~~
advanced networing
~~~

##### OS / ENVIRONMENT
~~~
vCenter
~~~

##### SUMMARY
I have a virtual switch configured in vCenter Switch0, associated with a physical nic. A port group has been created for virtual machines called LAN:

![image](https://github.com/apache/cloudstack/assets/60410678/dfbf55ee-c3ab-4b41-bbb9-862ba8730bff)

In cloudstack, when trying to import a VM, it appears as lan id 0, and it has been created with that id:

![image](https://github.com/apache/cloudstack/assets/60410678/d0c057a6-5b4b-483b-ace6-c285c202cced)

I'm sure I'm doing something wrong, that's why I'm asking for help.

##### STEPS TO REPRODUCE
~~~
When I click on import VM, the network does not detect me
~~~

##### EXPECTED RESULTS
~~~
Allow me to import the VM
~~~

##### ACTUAL RESULTS

![image](https://github.com/apache/cloudstack/assets/60410678/11adc8be-193e-4e4d-a5b5-d6c239d60787)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by shwstppr** [comment]
@ttmader I'm not sure but ACS UI might not be finding a matching network as the NIC for the VM returns a VLAN ID 0. Is it possible for you to test an import of a VM having a NIC with a different VLAN?

**Comment by shwstppr** [comment]
I'm not able to reproduce this with 4.19 pacakges

**Comment by DaanHoogland** [comment]
@ttmader , have you tried 4.19? Is this still an issue for you?

**Reply by DaanHoogland** [reply]
@ttmader , please investigate further and add more details if this is still an issue for you.

**Comment by DaanHoogland** [comment]
closed this for lack of interaction, please reopen or open a new item if applicable
